JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -104,7 +104,7 @@ private fun BottomBar(post: Post, onUnimplementedAction: () -> Unit) {
                         onBookmark = { toggleBookmark(postId = post.id) }
                     )
                     BottomBarAction(R.drawable.ic_share) {
-                        SharePost(post, context)
+                        sharePost(post, context)
```

Comment:
By convention we capitalize composable functions, please revert this.

Reply:
`sharePost` is not a composable function.

Reply:
Oh sorry!

Reply:
yayy! first PR in this repo lol. thx!